x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/cloud_security_posture_pli_auth_block_extension.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PropsWithChildren } from 'react';
+import React, { memo } from 'react';
+import { useUpsellingComponent } from '../../../../../common/hooks/use_upselling';
+
+export const CloudSecurityPosturePliAuthBlockExtension = memo<PropsWithChildren<unknown>>(
```

Comment:
I see why you placed this file and its lazy wrapper into this folder, but in general the files under `public/management` should be related to the 'management' pages (like Endpoints, Trusted Apps, Blocklists etc.)

What do you say about moving the shared dependencies (`FleetUiExtensionGetterOptions` from `types.ts` and `withSecurityContext`) up to a folder shared between `security_solution` teams (`public/common` maybe?), and then the two new files can be moved to a cloud security posture related folder?

Reply:
makes sense @gergoabraham, will push a fix